### ISSUE TITLE
Multiple select fields with more than 5 items should have an "unselect all"

### ISSUE THREAD

**Issue by justinscherer** [ushahidi/platform#2376]
## expected behaviour

Unselect all should behave according to web standards. If some of the list is selected and the user taps "unselect all" all become unselected, if some or none are unselected and user clicks "select all" all become selected.

## Current behaviour

<img width="565" alt="screen shot 2018-01-02 at 3 45 59 pm" src="https://user-images.githubusercontent.com/29209303/34503123-0d6c1302-efd4-11e7-8ffa-04d01d3058c5.png">

**Comment by EnriqueGalindo** [comment]
Hi, 
I'm interested in doing this as my first open source ticket. is it still available?

**Comment by EnriqueGalindo** [comment]
I forked and cloned the repo. Do you think you can tell me where I would be writing this?

**Comment by rowasc** [comment]
hi @EnriqueGalindo , I'm sorry I have missed this comment. Are you still interested in working on this issue?

**Comment by vandnakapoor19** [comment]
Hey @EnriqueGalindo  , I'm Outreachy 2020 Applicant and want to solve the issue.
Thanks

**Comment by rowasc** [comment]
@vandnakapoor19 I think Enrique has not progressed this task so you can go ahead and get started ! thank you for your interest in Ushahidi !

**Comment by rowasc** [comment]
@Obadha2 this needs QA (in steve)
Thanks!
